test/test_services_constructor.jl

```diff
@@ -195,7 +195,9 @@ end
     services = get_components(Service, c_sys5_uc)
     contributing_services = Vector{Service}()
     for service in services
-        push!(contributing_services, service)
+        if !(typeof(service) <: PSY.ReserveDemandCurve)
```

Comment:
Is there a way to express this behavior in a function?  If this is only code that will be used in this test then it's fine.  If not, this type of code doesn't grow well when new types are added.

Reply:
this is only for test, I don't expect it to be used anywhere else